src/frontend/components/InstructorDashboardVOD/index.spec.tsx

```diff
@@ -30,6 +30,24 @@ jest.mock('./widgets/InstructorDashboardVODWidgetGeneralTitle', () => ({
     <p>InstructorDashboardVODWidgetGeneralTitle</p>
   )),
 }));
+jest.mock('./widgets/InstructorDashboardVODWidgetUploadVideo', () => ({
+  InstructorDashboardVODWidgetUploadVideo: jest.fn(() => (
+    <p>InstructorDashboardVODWidgetUploadVideo</p>
+  )),
+}));
+jest.mock(
+  'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetThumbnail',
+  () => ({
+    DashboardVideoLiveWidgetThumbnail: jest.fn(() => (
+      <p>DashboardVideoLiveWidgetThumbnail</p>
+    )),
```

Comment:
```suggestion
    DashboardVideoLiveWidgetThumbnail: (props: { isLive: boolean }) => (
      <p>{`DashboardVideoLiveWidgetThumbnail ${props.isLive}`}</p>
    ),
```

There is no need to use `jest.fn` here

Reply:
Here I suggest you to display the `isLive` props to ensure it is called with the `false` value

Reply:
done